### PR TITLE
getting config of common costmap params to mirror strands_movebase

### PR DESCRIPTION
**Pull request by bfalacerda**

### CODE REVIEW

File: scitos_move_base_params/costmap_common_params.yaml [hunk 1]
```diff
@@ -3,9 +3,21 @@ robot_base_frame: /base_link
 
   
 map_type: voxel
-publish_voxel_map: true
-
-footprint: [[0.235, 0.31], [-0.515, 0.31], [-0.515, -0.31], [0.235, -0.31]]
+publish_voxel_map: false
```
Comment (nilsbore) [inline scitos_move_base_params/costmap_common_params.yaml]: This does not do anything if not within the namespace of the `object_layer`.

Comment (bfalacerda) [inline scitos_move_base_params/costmap_common_params.yaml]: you're right, and it looks there's not a map_type param at all, probably something we kept from groovy?

**Comment by bfalacerda** [comment]
i cant merge it though

**Comment by nilsbore** [comment]
Me neither=/.

**Comment by bfalacerda** [comment]
can someone give me and nils permissions for this repo? @hawesie @marc-hanheide

**Comment by marc-hanheide** [comment]
now you can

**Comment by bfalacerda** [comment]
cheers

**Pull request merged**